README.md

```diff
@@ -57,6 +60,11 @@ This library takes a set of different options whose expanded map is provided bel
       }
     });
 ```
+- `browser`: NOT OPTIONAL. Set to `true` for regular in-browser usage. `false` is used for debug purposes (prints results in the console).

```

Comment:
Just checking - why can't this just default to true? No big deal, just curious!

Reply:
Removed it altogether! No need for this now! Added a `query` option instead to satisfy issue #4

Reply:
Latest readme can be seen in the add-readme branch

here -> https://github.com/publiclab/matcher-core/blob/add-readme/README.md#arguments

Additionally, I've clarified a bit further on the constructor signature here -> https://github.com/publiclab/matcher-core/blob/add-readme/README.md#browser